src/core/http/filter.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Default, Clone, Debug)]
+/// User can configure the filter/interceptor
+/// for the http requests.
+pub struct HttpFilter {
+    pub on_request: Option<String>,
+}
```

Comment:
can delete this file. Move it to `http` module.

Reply:
Ok.